config.toml

```diff
@@ -56,76 +56,7 @@ timeout = 300000
     name = "Documentation"
     identifier = "docs"
     weight = 4
-
-[[menu.main]]
-    name = "Getting started"
-    parent = "docs"
-    weight = 1
-    identifier = "getting-started"
-    url = "/manual/latest/getting-started.html"
-
-[[menu.main]]
-    name = "User manual"
-    parent = "docs"
-    weight = 2
-    identifier = "manual"
-    url = "/manual/latest/"
-
-[[menu.main]]
-    name = "Component reference"
-    parent = "docs"
-    weight = 3
-    identifier = "components"
-    url = "/components/latest/"
-
-[[menu.main]]
-    name = "API Documentation"
-    parent = "docs"
-    weight = 4
-    identifier = "api-documentation"
-    url = "https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html" 
-
-[[menu.main]]
-    name = "Camel K"
-    parent = "docs"
-    weight = 5
-    identifier = "camel-k"
-    url = "/camel-k/latest/"
-
-[[menu.main]]
-    name = "Camel Quarkus"
-    parent = "docs"
-    weight = 6
-    identifier = "camel-quarkus"
-    url = "/camel-quarkus/latest/"
-
-[[menu.main]]
-    name = "Camel Kafka Connector"
-    parent = "docs"
-    weight = 7
-    identifier = "camel-kafka-connector"
-    url = "/camel-kafka-connector/latest/"
-
-[[menu.main]]
-    name = "Enterprise Integration Patterns"
-    parent = "docs"
-    weight = 8
-    identifier = "eip"
-    url = "/manual/latest/enterprise-integration-patterns.html"
-
-[[menu.main]]
-    name = "Camel 2.x to 3.0 Migration Guide"
-    parent = "docs"
-    weight = 9
-    identifier = "camel-3-migration-guide"
-    url = "/manual/latest/camel-3-migration-guide.html"
-
-[[menu.main]]
-    name = "Camel 3.x Upgrade Guide"
-    parent = "docs"
-    weight = 10
-    identifier = "camel-3x-upgrade-guide"
-    url = "/manual/latest/camel-3x-upgrade-guide.html"
+    url = "/documentations/"
```

Comment:
I'd go with:

```suggestion
    url = "/documentation.html"
```

Reply:
So I should not make a /documetations folder? We should just have a file?

Reply:
As we have it currently the `/documentations/` would not contain anything other than `index.html`, all the documentation we have is under `/manual/...`, `/components/...` and so on. I think it's better to have just `/documentation.html`.

Reply:
Shouldn't I be making a documentation.md file for content?

Reply:
I'd put this `_index.md` in `content/docs` and use the `layout/default/section.html` layout (which should be applied by default).

Reply:
On it!